compiler/docgen.nim

```diff
@@ -819,6 +821,18 @@ proc complexName(k: TSymKind, n: PNode, baseName: string): string =
       result.add(defaultParamSeparator)
       result.add(params)
 
+proc rstLinkName(k: TSymKind, n: PNode, baseName: string): string =
```

Comment:
links in lib/system_overview.rst (included from system.nim) work as intended, which is great; can you add a test to ensure links in includes keep working?

Reply:
ok, done, thank you for the idea!